pandas/tests/tseries/offsets/test_business_day.py

```diff
@@ -1,6 +1,8 @@
 """
 Tests for offsets.BDay
 """
+from __future__ import annotations
```

Comment:
Not clear why this is needed?

Reply:
for `type[BDay]`, would need to use `typing.Type[BDay]` otherwise

Reply:
I removed the `pandas/tests/tseries/offsets/test_*.py` changes